bindings/d/src/despacer.d

```diff
@@ -0,0 +1,19 @@
+module despacer.despacer;
+
+// idiomatic wrappers:
+
+// do not import libc as the extern C functions will conflict
+public import libc = despacer.libc;
+
+/**
+* remove spaces from string bytes (UTF-8 or ASCII)
+*/
+string despace(string str) nothrow
+{
+  import std : toStringz, fromStringz;
+
+  auto length = str.length;
+  auto cstr = cast(char*) toStringz(str.dup);
+  libc.despace(cstr, length);
```

Comment:
I get a missing reference error from the linker when I try to use this library in an executable.

```
Linking...
src/despacer.d:17: error: undefined reference to 'despace'
clang: error: linker command failed with exit code 1 (use -v to see invocation)
Error: /usr/bin/clang failed with status: 1
```